mlfinlab/tests/test_filters.py

```diff
@@ -37,7 +37,7 @@ def test_cusum_filter(self):
                 cusum_events = cusum_filter(self.data['close'], threshold=threshold, time_stamps=timestamp)
```

Comment:
Is this new method able to behave in the same way as the old one? 

I really like the idea here, its one I think we all wanted implemented when we started this project. Glad to see the change. I bring up the reference to the old funciton because the CUSUM filter is linked to Ch17 on structural breaks.

Reply:
Yeah, if fixed threshold is passed, behavior is the same. I didn't change your tests and passed them